cql3/statements/alter_table_statement.cc

```diff
@@ -398,19 +399,45 @@ alter_table_statement::prepare_schema_mutations(query_processor& qp, api::timest
 
 std::unique_ptr<cql3::statements::prepared_statement>
 cql3::statements::alter_table_statement::prepare(data_dictionary::database db, cql_stats& stats) {
+    // Cannot happen; alter_table_statement is never instantiated as a raw statement
+    // (instead we instantiate alter_table_statement::raw_statement)
+    abort();
```

Comment:
I see you just copied this from create_table_statement, so I'll accept it, but in the future, please consider using `utils::on_internal_error()` instead of abort. We'll get a useful message in the log, and one less chance to regret a crash we could have avoided.

Reply:
changed